playbooks/provisioning/openstack/post-provision-openstack.yml

```diff
@@ -90,3 +99,18 @@
         line: "IP4_NAMESERVERS={{ hostvars['localhost'].private_dns_server }}"
   roles:
     - node-network-manager
+
+- name: Show configured DNS nsupdate keys for future use as an external one
```

Comment:
Note, [this change](https://github.com/redhat-cop/infra-ansible/pull/63/files#diff-7e608771f779ebd52f40d3abf6fb8157R3) doesn't cover the specific case I encoded here. We need a tricky logic for putting the public_v4 as the private server entry, can't be used if dumped just as is...